### PR TITLE
Create milestones.md

### PR DESCRIPTION
**Pull request by pinorOSU**
Created visually appealing and informative forward-facing milestones document.

### CODE REVIEW

File: milestones.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    - [X] Libraries
+    - [X] Coding Standards
+- [x] 1. Having a running application list syntax required (any unordered or ordered list supported)
+    - [X] Fullscreen
+    - [X] Launches to game with no outside console commands
+    - [X] Menu may not be present
+- [x] 2. Rendering a single building
+    - [X] University
+    - [X] Reads info in from stock AOE2 files
+    -   -   [X] Internal only
```
Comment (heinezen) [inline milestones.md:20]: There must only be one `-` symbol, otherwise the final markdown looks weird.

**Comment by sandsmark** [comment]
The level of things is a bit arbitrary. At the level it is now there's a ton of things that could be added (garrisoning, terrain hill rendering, corpses, doppelgangers, automatic implicit technology research, random map scripts, scenario support, a ton of scenario triggers, campaign support, campaign cinematics, campaign cinematic crossfading, campaign, ai map analysis, animal behaviors, and I get tired just typing everything out..)

**Comment by TheJJ** [comment]
I think we need to "roll back" some changes now, since we need to re-design them with the new nyan converter and simulation code.

**Comment by pinorOSU** [comment]
All right- I sent an updated version a while ago from my new GH profile,
I'll take a look and make changes.

**Comment by heinezen** [comment]
@pinorOSU Any news on this?

**Comment by TheJJ** [comment]
The current in-tree milestones-file is very sparse, but reflects the current state:
https://github.com/SFTtech/openage/blob/master/doc/milestones.md

I'm not so sure how we should extend it with more steps, since we implement stuff just on demand and have to react to many things we couldn't foresee. What do you think how we should design the milestones file?

**Comment by TheJJ** [comment]
Yo @pinorOSU, i'll close this due to inactivity for now, but please ping back if I should reopen the pull!

**Pull request closed without merging**